packages/create-catalyst/README.md

```diff
@@ -5,13 +5,13 @@ Create a new Catalyst project, optionally connect the project to a BigCommerce s
 ## NPM
 
 ```sh
-npm create catalyst-storefront@latest --storeHash your_store_hash --accessToken your_access_token
+npm create catalyst-storefront@latest
 ```
 
 ## PNPM
 
 ```sh
-pnpm create catalyst-storefront@latest --storeHash your_store_hash --accessToken your_access_token
+pnpm create catalyst-storefront@latest
```

Comment:
Now that the library prompts for store hash and access token (and adds some helpful logs instructing the user how to create those tokens), we should discourage usage of flags for authentication, and use them solely for CI runs.